content/accessibility/accessibility-at-github.mdx

```diff
@@ -0,0 +1,48 @@
+
+## What standards you must aim for 
+GitHub aims for [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/) AA compliance. This includes all of WCAG 2.0 AA plus additional considerations. 
+
+Currently, [Section 508](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/overview-of-the-final-rule) (required by law in the United States) follows WCAG 2.0 AA guidelines for compliance. 
+
+## Who should you consider when designing? 
+There are a whole host of disabilities to consider, though the majority will fall into one of these categories: 
+- **Visual:** Anything that deals with sight, or technology that helps someone to see. This includes people who have different types of sight like color blindness.
+- **Cognitive:** Concentration, memory, judgement, problem solving, logic skills. 
+- **Mobility:** Anything that affects movement in a body. 
+- **Hearing:** A spectrum of disabilities related to sound or audio.
+
+Each of these disability categories can be divided into three further sub-categories:
+- **Permanent:** Disability does not go away.
+  - Example: Someone who is deaf
+- **Temporary:** The disability will go away in time.
+  - Example: Someone who has an ear infection
+- **Situational:** The environment someone is in plays a huge factor in how they interact with web content. 
+  - Example: Someone who is at a loud sporting event
+
+For an informative list on real-life disability situations, check [An Alphabet of Accessibility Issues](https://the-pastry-box-project.net/anne-gibson/2014-july-31), published in The Pastry Box Project.
+
+Microsoft has created a downloadable PDF that looks into the area of inclusive design from a people perspective, with several informative examples. Navigate to the ‘Inclusive 101’ toolkit on the main page of [Microsoft’s Inclusive Design website](https://www.microsoft.com/design/inclusive/).
+
+## Internal resources
+If you're GitHub staff and need help with accessibility:
+- Visit the #accessibility Slack channel to ask questions or discuss accessibility issues
+- Check [github/accessibility](https://github.com/github/accessibility) repository for information on events or learning resources. 
+- Attend the inclusive design office hours with @ichelsea on Tuesdays (the time alternates between  10am PST and 1pm PST every other week)
+
+## Support
+Accessibility is a priority for GitHub. If you ever encounter accessibility related issues when using github.com, please don’t hesitate to get in touch via [the contact page](https://support.github.com/contact) or email [support@github.com](support@github.com) with your concerns.
+
+For information about the accessibility compliance of GitHub products, please refer to the [VPAT report, outlining §508 accessibility information for GitHub.com, GitHub Enterprise, and GitHub Desktop](https://government.github.com/accessibility/).
```

Comment:
Shouldn't this be written under the assumption that we are speaking to internal folks already? The last support section also feels more in line with what should be in our GitHub product support pages, not necessarily in our design docs.

Reply:
I wonder if the purpose is in case a user lands on this page, and is looking for support?

Reply:
> I wonder if the purpose is in case a user lands on this page, and is looking for support?

Yep, I can envision anyone inside of GitHub that's curious about our accessibility efforts on the design team landing here.